controller/src/beerocks/master/db/db.h

```diff
@@ -512,6 +512,17 @@ class db {
     bool set_channel_scan_pool(const sMacAddr &mac, const std::unordered_set<uint8_t> &channel_pool,
                                bool single_scan);
 
+    /**
+     * @brief Validate the channel scan pool
+     * 
+     * @param mac:          MAC address of radio
+     * @param channel_pool: Channel pool of channel scan
+     * @return true if pool is valid
+     * @return false if pool is invalid
```

Comment:
To me the function name is pretty explicit about what is being done already, but the documentation is so vague that it doesn't bring anything useful. 
What kind of validation does it do?

For example:

```suggestion
    /**
     * @brief Validate the channel scan pool.
     * 
     * A pool is considered valid if all the channels it contains are supported by the radio.
     * 
     * @param mac:          MAC address of the radio
     * @param channel_pool: Channel pool to validate
     * @return true if channel_pool is valid, false otherwise.
```

(nitpick)

Reply:
Updated the doxygen documentation